BaseUI/src/components/SubscriberCell.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {View, Image, Text} from 'react-native';
+import SubscriberCellProps from '../interfaces/SubscriberCellProps';
+//import SubscriberCellStyle from '../styles/SubscriberCellStyle';
+import FilledButton from './FilledButton';
+// import {CheckBox} from 'react-native-elements';
+// import Icon from 'react-native-vector-icons/FontAwesome';
+// import {CheckBox} from '@rneui/base';
+import CheckBox from '@react-native-community/checkbox';
+
+// Icon.loadFont();
+
+const SubscriberCell = (props: SubscriberCellProps) => {
+  //console.log(props.subscriber);
+  return (
+    <View style={props.styles.subscriberCellContainerStyle}>
+      <View style={props.styles.imageStyle}>
+        <Image
+          source={{
+            uri: 'file:///Users/gergely-csepanyi/Documents/Projects/04_BasicUIComponents/BaseUI/src/images/profileImagePlaceholder.jpg',
+          }}
+        />
+      </View>
+      <View style={props.styles.subscriberTextsContainer}>
+        <Text style={props.styles.titleTextStyle}>{props.subscriber.name}</Text>
+        <Text style={props.styles.descriptionTextStyle}>
+          {props.subscriber.description}
+        </Text>
+      </View>
+
+      {props.renderButtonOrCheckbox === 'button' && (
```

Comment:
'renderButtonOrCheckbox' it's better to name it as 'isButton' or 'renderComponentType'